Frontend/library/src/WebRtcPlayer/WebRtcPlayerController.ts

```diff
@@ -1487,6 +1487,44 @@ export class WebRtcPlayerController {
         }
     }
 
+    /**
+     * Send the MinQuality encoder setting to the UE Instance.
+     * @param minQuality - The lower bound for quality when encoding
+     * valid values are (0-100) where:
+     * 0 = Worst quality.
+     * 100 = Best quality.
+     */
+    sendEncoderMinQuality(minQuality: number) {
+        Logger.Info(`MinQuality=${minQuality}\n`);
+
+        if (minQuality != null) {
+            this.streamMessageController.toStreamerHandlers.get('Command')([
+                JSON.stringify({
+                    'Encoder.MinQuality': minQuality
+                })
+            ]);
+        }
+    }
+
+    /**
+     * Send the MaxQuality encoder setting to the UE Instance.
+     * @param maxQuality - The upper bound for quality when encoding
+     * valid values are (0-100) where:
+     * 0 = Best quality.
+     * 100 = Worst quality.
```

Comment:
Shouldn't this and one above be 0 = Worst quality, 100 = Best quality.'

Reply:
Probably